### PR TITLE
Design Fixed

### PR DESCRIPTION
**Pull request by gowtham-calib**

### CODE REVIEW

File: frontend/src/app/components/Launchpad/nftStorageService.ts [hunk 1]
```diff
@@ -10,7 +9,7 @@ const client = new NFTStorage({
 
 function getImageBlob(data: LaunchFormData): File | Blob {
   const toFile = (file: RcFile) => {
-    const [, extension] = file.type.split("/");
+    const [extension] = file.type.split("/");
```
Comment (nandhabn) [inline frontend/src/app/components/Launchpad/nftStorageService.ts:12]: here we only need the second param only. so please revert this change.

**Review by Yazhinivenkatesh — APPROVED** [review]

**Review by nandhabn — CHANGES_REQUESTED** [review]

File: frontend/src/app/components/Launchpad/nftStorageService.ts (hunk 1 above)
Comment (gowtham-calib) [reply frontend/src/app/components/Launchpad/nftStorageService.ts]: Updated the changes

File: frontend/src/app/components/Header/index.scss [hunk 2]
```diff
@@ -1,7 +1,7 @@
 @import "../../../assets/styles/settings/colors";
 
 .header {
-  background-color: var(--primary-blue);
+  background-color: var(--header-color);
```
Comment (nehaannvinu) [inline frontend/src/app/components/Header/index.scss:4]: As per design, header does not have a different color

**Review by nandhabn — APPROVED** [review]
